app/components/SøknadFooter.tsx

```diff
@@ -0,0 +1,24 @@
+import { Button, HStack } from "@navikt/ds-react";
+import { FloppydiskIcon } from "@navikt/aksel-icons";
+import { SlettSøknadSpørsmålModal } from "~/components/SlettSøknadSpørsmålModal";
+
+interface SøknadFooterProps {
+  soknadId: string;
+}
+
+export function SøknadFooter({ soknadId }: SøknadFooterProps) {
+  return (
+    <div className="innhold">
+      <HStack>
+        <Button
+          variant="tertiary"
+          size="small"
+          icon={<FloppydiskIcon title="a11y-title" fontSize="1.5rem" />}
+        >
+          Fortsett senere
+        </Button>
+        <SlettSøknadSpørsmålModal søknadId={soknadId} />
```

Comment:
Det blir kanskje litt rar logikk, men hadde det vært bedre med en knapp her? Jeg tenkte det kanskje er lettere å lese koden hvis det er "knapp - knapp", i stedet for "knapp - modal (med knapp inni, men det vet man ikke før man åpner den koden)". Men da må man kanskje ha mer context-greier? 🤔